src/basics.php

```diff
@@ -51,7 +51,13 @@ function debug($var, $showHtml = null, $showFrom = true)
         $line = '';
         $lineInfo = '';
         if ($showFrom) {
-            $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+            $trace = Debugger::trace(['start' => 1, 'depth' => 3, 'format' => 'array']);
+            if (count($trace) > 1
+                && $trace[1]['function'] === 'dd'
```

Comment:
The current implementation isn't debug() + die() though. Its pr() + die(). We could rename the dd() function as an alternative.